### ISSUE TITLE
Tests are broken

### ISSUE THREAD

**Issue by rvillablanca** [aeperez24/videoScrapper#3]
```
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
        panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0x4851d4]

```

**Comment by rvillablanca** [comment]
@aeperez24 

```
?   	aeperez24/videoScrapper	[no test files]
?   	aeperez24/videoScrapper/application	[no test files]
?   	aeperez24/videoScrapper/mock/animeshow	[no test files]
?   	aeperez24/videoScrapper/mock/port	[no test files]
?   	aeperez24/videoScrapper/mock/service	[no test files]
?   	aeperez24/videoScrapper/port	[no test files]
2022/08/30 08:07:01 downloading fromlocation
--- FAIL: TestDownloadFromLink (0.00s)
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
	panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0x4851d4]

goroutine 9 [running]:
testing.tRunner.func1.2({0x6170a0, 0x801510})
	/home/rodrigo/go/go1.18.3/src/testing/testing.go:1389 +0x24e
testing.tRunner.func1()
	/home/rodrigo/go/go1.18.3/src/testing/testing.go:1392 +0x39f
panic({0x6170a0, 0x801510})
	/home/rodrigo/go/go1.18.3/src/runtime/panic.go:838 +0x207
io.ReadAll({0x0, 0x0})
	/home/rodrigo/go/go1.18.3/src/io/io.go:645 +0xd4
io/ioutil.ReadAll(...)
	/home/rodrigo/go/go1.18.3/src/io/ioutil/ioutil.go:27
aeperez24/videoScrapper/provider/animeshow.TestDownloadFromLink(0x0?)
	/home/rodrigo/dev/github/videoScrapper/provider/animeshow/downloader_service_test.go:68 +0x97
testing.tRunner(0xc0001e4340, 0x670ca8)
	/home/rodrigo/go/go1.18.3/src/testing/testing.go:1439 +0x102
created by testing.(*T).Run
	/home/rodrigo/go/go1.18.3/src/testing/testing.go:1486 +0x35f
FAIL	aeperez24/videoScrapper/provider/animeshow	0.009s
ok  	aeperez24/videoScrapper/provider/cuevana	(cached)
ok  	aeperez24/videoScrapper/service	(cached)
FAIL
```

**Comment by rvillablanca** [comment]
I think the error is right here https://github.com/aeperez24/videoScrapper/blob/77e234f0bce5f8f5f97da6466e54d7ea5d892e54/provider/animeshow/downloader_service_test.go#L83, it should be `episodeMu.html` and not `episodeMU.html` (uppercase U).

This was particularly difficult to find because there are a lot of error ignored there in the tests, for instance, the `open` function in `downloader_service_test.go` the `err` returned by `os.Open` is ignored.

**Comment by rvillablanca** [comment]
Fixed by #5
